prepare/envsetup.sh

```diff
@@ -406,6 +406,28 @@ local_setup_configure_nm_inf () {
     touch /tmp/inf_configured
 }
 
+install_usb_hub_driver () {
+    # Works under RHEL 8.0.
+    yum install -y libffi-devel python36-devel
+    pushd /tmp
+        wget https://acroname.com/system/files/software/brainstem_dev_kit_ubuntu_lts_18.04_no_qt_x86_64_1.tgz
+        tar xf brainstem_dev_kit_ubuntu_lts_18.04_no_qt_x86_64_1.tgz
+        cd development/python/
+        python3 -m pip install brainstem-2.7.0-py2.py3-none-any.whl; local rc=$?
+    popd
+    return $rc
+}
+
+install_usb_hub_utility () {
+    # Download utility for Acroname USB hub - CLI.
+    # git clone https://github.com/rcorreia/acroname-python-cli.git
+    # Using a local copy of that utility.
+    mkdir /tmp/acroname-python-cli
+    chmod +x prepare/acroname.py
```

Comment:
What about use tmp dir directly? in NM source I mean, w/o all this copying magic, just PYTHON_PATH env needs to be set. Something like $(pwd)/tmp, but this is not compulsory, feel free to leave it as you need it, but please use tmp instead of prepare.